pkg/virt-handler/vm_test.go

```diff
@@ -1847,6 +1847,44 @@ var _ = Describe("VirtualMachineInstance", func() {
 			testutils.ExpectEvent(recorder, VMIMigrating)
 		})
 
+		It("should not try to migrate a vmi twice", func() {
```

Comment:
I would add one more unit test that will verify second different migration is allowed. WDYT?

Reply:
discussed offline, we already have an e2e test that does that.
`[release-blocker][test_id:1377]should be successfully migrated multiple times`